mne/epochs.py

```diff
@@ -583,7 +583,7 @@ def _detrend_offset_decim(self, epoch, verbose=None):
         return epoch
 
     def iter_evoked(self):
-        """Iterate over Evoked objects with nave=1
+        """Iterate epochs as a sequence of Evoked objects with n_epochs=1
```

Comment:
`nave` is an attribute of `evoked` object. I feel you should keep that but add in brackets that it means only one epoch has been averaged ...